ssl/ssl_lib.c

```diff
@@ -5399,6 +5399,34 @@ int SSL_client_hello_get1_extensions_present(SSL *s, int **out, size_t *outlen)
     return 0;
 }
 
+size_t SSL_client_hello_get1_raw_extensions(SSL *s, unsigned short *extensions)
```

Comment:
Again `get1` is incorrect here.  I'm not sure if it should be `get0` or just `get`.  I'm leaning towards `get` since it is accepting a pointer to application controlled memory and `get1` and `get0` both seem wrong.

Reply:
Yeah, it should be just `get` here.

Reply:
rename to SSL_client_hello_get_raw_extensions